### ISSUE TITLE
Allow the user to force multiple dictionaries for all sites from those installed in the browser

### ISSUE THREAD

**Issue by Korb** [kimsey0/FirefoxAutoDict#13]
It's strange, in Firefox 113.0.1 (64-bit) only one dictionary is activated by default, and sharing multiple dictionaries is possible only after manually activating them for each site where the user types texts.

**Comment by dreua** [comment]
I totally agree but this might not be the right place to ask/discuss this. Let me know if you find a solution anyway :wink: 

I think this addon is more or less obsolete once this issue is fixed you can use multiple dictionaries seamlessly on all sites.
